.github/workflows/linting.yaml

```diff
@@ -1,7 +1,7 @@
 name: Linting
 on:
-  pull_request_target:
-    branches: [main, ga-staging]
+  pull_request:
```

Comment:
```suggestion
  pull_request_target:
```

Reply:
we still need `_target` in this case because the workflow needs access to the `GITHUB_TOKEN` secret.